.vitepress/theme/components/VersionReminder.vue

```diff
@@ -21,7 +21,7 @@ const latest = computed(() => {
 
 const old = computed(() => {
   var versionSplit = version.value.split(".");
-  if (versionSplit[1] <= 21 && versionSplit[2] <= 10) {
+  if (parseInt(versionSplit[1]) <= 21 && parseInt(versionSplit[2]) < 10) {
```

Comment:
```suggestion
  if (parseInt(versionSplit[0]) === 1 && parseInt(versionSplit[2]) < 10) {
```

should work all the same (coz fabric doesn't support 1.7.10), and not break with the new numbering.